<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>432</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Injection Clicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_F1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1111</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1122</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1251</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1520</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_F8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>24522</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_F7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time interval (ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_F7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set PID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PID">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_PID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PID_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_PID_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tittle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Window Handler</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Keys to inject</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Contact Author: Napalys.Klicius@gmail.com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>checkBox_F1</zorder>
   <zorder>lineEdit_F1</zorder>
   <zorder>label_F1</zorder>
   <zorder>checkBox_F2</zorder>
   <zorder>label_F2</zorder>
   <zorder>lineEdit_F2</zorder>
   <zorder>checkBox_F3</zorder>
   <zorder>checkBox_F4</zorder>
   <zorder>lineEdit_F4</zorder>
   <zorder>label_F4</zorder>
   <zorder>label_F3</zorder>
   <zorder>lineEdit_F3</zorder>
   <zorder>checkBox_F5</zorder>
   <zorder>checkBox_F6</zorder>
   <zorder>lineEdit_F6</zorder>
   <zorder>label_F6</zorder>
   <zorder>label_F5</zorder>
   <zorder>lineEdit_F5</zorder>
   <zorder>checkBox_F7</zorder>
   <zorder>checkBox_F8</zorder>
   <zorder>lineEdit_F8</zorder>
   <zorder>label_F8</zorder>
   <zorder>label_F7</zorder>
   <zorder>lineEdit_F7</zorder>
   <zorder>pushButton_PID</zorder>
   <zorder>lineEdit_PID</zorder>
   <zorder>label_PID</zorder>
   <zorder>lineEdit_PID_2</zorder>
   <zorder>label_PID_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_Start</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>238</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
